README.rst

```diff
@@ -51,6 +51,7 @@ Usage
   $ vulture myscript.py mypackage/
   $ vulture myscript.py mywhitelist.py  # Put false-positives in whitelist.
   $ vulture myscript.py --min-confidence 100  # Only report 100% dead code.
+  $ vulture myscript.py --make-whitelist  # Report in whitelist format.
```

Comment:
Can you move this example to a dedicated section that explains how and why to use the new feature, please?

Reply:
Done!